nmdc_schema/migration_recursion.py

```diff
@@ -154,12 +158,12 @@ def main(schema_path, input_path, output_path, salvage_prefix, migrator_name):
     curie_migrator = CurieMigrator()
     curie_migrator.forced_prefix = salvage_prefix
 
-    # iterate over collections, applying CURIe fixes
-    for tdk, tdv in total_dict.items():
-        logger.info(f"Fixing CURIes in {tdk} if necessary")
-
-        total_dict[tdk] = curie_migrator.fix_curies_recursively_by_key(
-            tdv, set(curie_slots))
+    # # iterate over collections, applying CURIe fixes
+    # for tdk, tdv in total_dict.items():
+    #     logger.info(f"Fixing CURIes in {tdk} if necessary")
+    #
+    #     total_dict[tdk] = curie_migrator.fix_curies_recursively_by_key(
+    #         tdv, set(curie_slots))
```

Comment:
If you are keeping this only so you can easily restore it (and not so you can read it when reading the rest of the code), I'd prefer it be deleted. It will still be available in the Git history.